src/ui/public/doc_table/doc_table.html

```diff
@@ -20,6 +20,12 @@
           class="discover-table-row"></tr>
       </tbody>
     </table>
+    <div ng-if="hits.length === sampleSize && page.last" class="discover-table-footer">
```

Comment:
We can't tell the user how many rows are hidden when using this from dashboard because we don't have the total number of results in that case, we are only passing searchSource.

Discover won't ever show this part because it uses infiniteScroll and passes hits instead of using searchSource & setting hits.  I agree it's unintuitive that `hits` is two different things depending on infiniteScroll and searchSource, but I don't want to get into a refactor since this is a Mend it Monday `low fruit` issue and if I make larger modifications it should probably go through testing.  Thoughts?